test/data/testTD.py

```diff
@@ -185,6 +190,10 @@ def testsecs_transition(self):
         """ROCIS does not form singly excited configurations (secs)"""
         pass
 
+    def testrotatsnumber(self):
+        """ROCIS does not calculate rotatory strengths"""
```

Comment:
Is this specific to GAMESS?

Reply:
Expand the context (that's a GitHub bug I think), it's specific to ORCA.